lib/stats.js

```diff
@@ -56,7 +56,7 @@ function get (pids, options, callback) {
     }
   }
 
-  stat(pids, {}, function (err, stats) {
+  stat(pids, options, function (err, stats) {
```

Comment:
Add JSDoc for options also inside index.js
https://github.com/simonepri/pidtree/blob/master/index.js#L20